api/src/main/java/io/strimzi/api/kafka/model/KafkaJmxOptionsAuthentication.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.api.kafka.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonSubTypes;
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+import io.strimzi.crdgenerator.annotations.Description;
+import lombok.EqualsAndHashCode;
+
+import java.io.Serializable;
+import java.util.HashMap;
+import java.util.Map;
+
+/*
+ * Configure the authentication protocol on the Kafka Broker's JMX port.
+ */
+@JsonTypeInfo(use = JsonTypeInfo.Id.NAME,
+        include = JsonTypeInfo.As.EXISTING_PROPERTY,
+        property = "type")
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@EqualsAndHashCode
+@JsonSubTypes({
+        @JsonSubTypes.Type(name = KafkaJmxOptionsAuthenticationSimple.TYPE_SIMPLE, value = KafkaJmxOptionsAuthenticationSimple.class)
+})
+public abstract class KafkaJmxOptionsAuthentication implements UnknownPropertyPreserving, Serializable {
+    private static final long serialVersionUID = 1L;
+    private Map<String, Object> additionalProperties = new HashMap<>(0);
+
+    @Description("Authentication type. " +
+            "Currently the only supported types are `plain`." +
+            "`plain` type creates a username and protected port with no TLS.")
```

Comment:
We should fix the description: the type is now `simple`.

Reply:
Agreed